bip-0360.mediawiki

```diff
@@ -11,696 +11,386 @@
 |-
-! Type !! Vulnerable !! Prefix !! Example
+! Type
+! Vulnerable
+! Prefix
+! Example
 |-
-| P2PK || Yes || Varies || 2103203b768951584fe9af6d9d9e6ff26a5f76e453212f19ba163774182ab8057f3eac
+| P2PK
+| Yes
+| Varies
+| 2103203b768951584fe9af6d9d9e6ff26a5f76e453212f19ba163774182ab8057f3eac
 |-
-| P2PKH || No¹ || 1 || 1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa
+| P2PKH
+| No¹
+| 1
+| 1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa
 |-
-| P2MS || Yes || Varies || 52410496ec45f878b62c46c4be8e336dff7cc58df9b502178cc240e...
+| P2MS
+| Yes
+| Varies
+| 52410496ec45f878b62c46c4be8e336dff7cc58df9b502178cc240e…
 |-
-| P2SH || No¹ || 3 || 3FkhZo7sGNue153xhgqPBcUaBsYvJW6tTx
+| P2SH
+| No¹
+| 3
+| 3FkhZo7sGNue153xhgqPBcUaBsYvJW6tTx
 |-
-| P2WPKH || No¹ || bc1q || bc1qsnh5ktku9ztqeqfr89yrqjd05eh58nah884mku
+| P2WPKH
+| No¹
+| bc1q
+| bc1qsnh5ktku9ztqeqfr89yrqjd05eh58nah884mku
 |-
-| P2WSH || No¹ || bc1q || bc1qvhu3557twysq2ldn6dut6rmaj3qk04p60h9l79wk4lzgy0ca8mfsnffz65
+| P2WSH
+| No¹
+| bc1q
+| bc1qvhu3557twysq2ldn6dut6rmaj3qk04p60h9l79wk4lzgy0ca8mfsnffz65
 |-
-| P2TR || Yes || bc1p || bc1p92aslsnseq786wxfk3ekra90ds9ku47qttupfjsqmmj4z82xdq4q3rr58u
+| P2TR
+| Yes
+| bc1p
+| bc1p92aslsnseq786wxfk3ekra90ds9ku47qttupfjsqm mj4z82xdq4q3rr58u
 |-
-| P2QRH || No || bc1z || bc1zzmv50jjgxxhww6ve4g5zpewrkjqhr06fyujpm20tuezdlxmfphcqfc80ve
+| P2TSH
+| No¹
+| bc1z
+| bc1zzmv50jjgxxhww6ve4g5zpewrkjqhr06fyujpm20tuezdlxmfphcqfc80ve
 |}
+¹ funds in P2PKH, P2SH, P2WPKH, P2WSH, and P2TSH outputs become vulnerable to long-exposure quantum attacks anytime their redeem script is revealed.
 
-¹ Funds in P2PKH, P2SH, P2WPKH, and P2WSH outputs become vulnerable to long-exposure quantum attacks when their input script is revealed. An address is no longer safe against long-exposure quantum attacks after funds from it have been spent.
-
-It should be noted that Taproot outputs are vulnerable in that they encode a 32-byte x-only public key, from which a
-full public key can be reconstructed.
-
-If a CRQC recovers an extended public key (xpub), including its chain code, it can derive all non-hardened child public
-keys by guessing or iterating through child indexes, as allowed by BIP 32's non-hardened derivation. With Shor's
-algorithm, the CRQC could then compute the corresponding non-hardened child private keys directly from those public keys,
-without needing the extended private key (xprv) or an exposed child private key. Hardened child keys remain secure since
-they cannot be derived from the xpub alone. However, if the xprv is exposed, then all child private keys--both hardened
-and non-hardened--become vulnerable. Thus, in a quantum context, the xpub alone is sufficient to expose all non-hardened
-child private keys.
+A complete list of outputs that are vulnerable to long-exposure attacks:
 
-==== Long Exposure and Short Exposure Quantum Attacks ====
+* Legacy (i.e. P2PK) outputs (Satoshi’s coins, CPU miners)
+* Reused outputs of any type
```

Comment:
```suggestion
* Reused addresses (any type, except P2TSH)
```

Reply:
This is not even true -- re-using a P2TSH output is as vulnerable as re-using any other output type. This is why you never re-use outputs once spent. 

These changes would also reintroduce the incorrect terminology.